src/features/home/pages/helpers/useOnScroll.ts

```diff
@@ -3,19 +3,23 @@ import { NativeScrollEvent, NativeSyntheticEvent, Platform } from 'react-native'
 import { useOpacityTransition } from 'ui/animations/helpers/useOpacityTransition'
 import { useDebounce } from 'ui/hooks/useDebounce'
 
-export const useOnScroll = (listener: (e: NativeSyntheticEvent<NativeScrollEvent>) => void) => {
-  const debouncedScrollListener = useDebounce(listener, 100)
-  const scrollListener = (e: NativeSyntheticEvent<NativeScrollEvent>) => {
+export const useOnScroll = (
+  debounceListener: (e: NativeSyntheticEvent<NativeScrollEvent>) => void,
+  scrollListener: (e: NativeSyntheticEvent<NativeScrollEvent>) => void
+) => {
+  const debouncedAnalyticsListener = useDebounce(debounceListener, 100)
+  const Listeners = (e: NativeSyntheticEvent<NativeScrollEvent>) => {
```

Comment:
Pourquoi une majuscule ? C'est juste une fonction : 
```suggestion
  const listeners = (e: NativeSyntheticEvent<NativeScrollEvent>) => {
```